Source/WebKit/UIProcess/API/Swift/WebView.swift

```diff
@@ -103,7 +104,16 @@ fileprivate struct WebViewRepresentable {
     }
 
     func updatePlatformView(_ platformView: WebViewWrapper, context: Context) {
-        platformView.webView = owner.page.backingWebView
+        let webView = owner.page.backingWebView
+        let environment = context.environment
+
+        platformView.webView = webView
+
+        webView.allowsBackForwardNavigationGestures = environment.webViewAllowsBackForwardNavigationGestures
```

Comment:
Are we checking somewhere whether these values are actually changes. As in, this won’t kick off unnecessary work? This method could be called semi-frequently (mentally I’d categorize it like `-layoutSubviews` even though that’s not exactly true), so we should just be mindful of performance is all.

Reply:
the underlying setters in WKWebView have early returns in the case where the values don't change